tests/test_tox_uv_venv.py

```diff
@@ -50,6 +51,38 @@ def test_uv_venv_spec_major_only(tox_project: ToxProjectCreator) -> None:
     result.assert_success()
 
 
+@pytest.mark.parametrize(
```

Comment:
Conflict test should be separate, and you did not assert failure message 🤔

Reply:
Done